packages/ubuntu_init/lib/src/launchsession/launchsession_page.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:ubuntu_wizard/ubuntu_wizard.dart';
+import 'package:yaru_widgets/yaru_widgets.dart';
+
+import 'launchsession_l10n.dart';
+import 'launchsession_model.dart';
+
+class LaunchSessionPage extends ConsumerWidget {
+  const LaunchSessionPage({super.key});
+
+  static Future<bool> load(WidgetRef ref) {
+    return ref.read(launchsessionModelProvider).init();
+  }
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final l10n = LaunchsessionLocalizations.of(context);
+    return WizardPage(
+      title: YaruWindowTitleBar(
+        title: Text(l10n.storeTitle),
+      ),
+      content: Column(
+        crossAxisAlignment: CrossAxisAlignment.center,
+        children: [
+          Expanded(
+            child: SvgPicture.asset('assets/store.svg', package: 'ubuntu_init'),
```

Comment:
Also probably shouldn't use the store image

Reply:
Fixed.